tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -153,113 +153,113 @@
-    <string name="pref_fast_block_http_summ">Deny connection to http port 80</string>
-    <string name="pref_fast_categ_update">Update</string>
-    <string name="pref_fast_auto_update">Automatic Updates</string>
-    <string name="pref_fast_auto_update_summ">Check InviZible Pro and modules Updates daily</string>
-    <string name="pref_fast_chek_update">Check Update</string>
-    <string name="pref_fast_chek_update_summ">Check if new versions are available now</string>
-    <string name="pref_fast_through_tor_update">Update strictly through Tor</string>
-    <string name="pref_fast_through_tor_update_summ">Check updates only with Tor and when Tor is running</string>
-
-    <string name="pref_common_categ_run_with_root">Use Root Privileges</string>
+    <string name="pref_fast_tor_bypass_lan_summ">Do not use Tor for LAN destinations and IANA reserved IP blocks.</string>
+    <string name="pref_fast_language">Language</string>
+    <string name="pref_fast_block_http">Block HTTP</string>
+    <string name="pref_fast_block_http_summ">Deny HTTP connection over port 80.</string>
+    <string name="pref_fast_categ_update">New Versions</string>
+    <string name="pref_fast_auto_update">Auto-installation</string>
+    <string name="pref_fast_auto_update_summ">Check for new versions of InviZible Pro and modules daily.</string>
+    <string name="pref_fast_chek_update">Look for new versions</string>
+    <string name="pref_fast_chek_update_summ">Check if new versions are available now.</string>
+    <string name="pref_fast_through_tor_update">Download any new versions only through Tor</string>
+    <string name="pref_fast_through_tor_update_summ">Look for new versions only with Tor and when Tor is running.</string>
+
+    <string name="pref_common_categ_run_with_root">Use Root privileges</string>
     <string name="pref_common_run_with_root">Run Modules With Root</string>
-    <string name="pref_common_run_with_root_summ">Use Root privileges for DNSCrypt, Tor and I2P modules.
-        Enabling this feature leaves modules unmanaged and may cause connection problems!</string>
-    <string name="pref_common_categ_other">Other</string>
-    <string name="pref_common_block_http">Block HOTSPOT http</string>
-    <string name="pref_common_block_http_summ">Deny connection to http port 80 for HOTSPOT</string>
-    <string name="pref_common_notification_summ">Protect application with No Root Mode to prevent app being killed by android</string>
-    <string name="pref_common_mitm_categ">MITM attack detection</string>
-    <string name="pref_common_arp_spoofing_detection">ARP spoofing detection</string>
+    <string name="pref_common_run_with_root_summ">Use root privileges for DNSCrypt, Tor and I2P modules.
+        Using this feature leaves modules unmanaged and may cause connection problems!</string>
+    <string name="pref_common_categ_other">Misc</string>
+    <string name="pref_common_block_http">Block HOTSPOT HTTP</string>
```

Comment:
Is this "Wi-Fi hotspot" from [line 729](https://github.com/Gedsh/InviZible/pull/111/files#diff-5a76a99c2acfa621a06d3271e89047bea02a6126abf571c3b154fc835c668d77R729)?

Reply:
This hotspot starts locally on the device. Hotspot on line 729 means a specific Wi-Fi network.

Reply:
729 I understand. This I don't.
Is it:
~I am blocking HTTP traffic on the mobile Wi-Fi hotspot that is or could be running?